src/classmentors/components/events/events.js

```diff
@@ -2112,7 +2111,7 @@ function ClmEventTableCtrl($scope, $q, $log, $mdDialog, $document,
         var parent = this;
 
         function loadEditor() {
-            var editor = ace.edit($document.querySelector('#editor'));
+            var editor = ace.edit(document.querySelector('#editor'));
```

Comment:
Is it possible to use the inject to find the element instead global, which will prevent you to test it.

Also, it's not important for this PR but you should avoid manipulate the DOM from a controller. You should either create a component / directive to handle this, or reuse [spfEditor](https://github.com/singpath/classmentors/blob/master/src/packages/singpath-core/src/components/ace/ace.js) component.